### DISCUSSION TITLE
Welcome to Nap Discussions!

### DISCUSSION THREAD

**Discussion by maaslalani** [maaslalani/nap#1] [Announcements]
## Welcome to Nap Discussions

Feel free to ask any questions about Nap and suggests new features that you would like to see implemented!

**Comment by wiredolphin** [comment]
Hi. As foreward I just want to thank you all for the nice work, Nap is really an useful tool for me as developer (actually not in the Go world). 
I just want to ask If there is actually any plan to make the snippets folder to be rearranged/sorted in some way. This could be helpful in case of a long list of folders. 
I'we noticed that the `snippet.json` file will be automatically recreated when Nap is restarted, so any change to it hasn't any effect on the order of the snippets/folders, unless I'm missing something.

**Reply by maaslalani** [reply]
Hey @wiredolphin does this do what you're asking for? https://github.com/maaslalani/nap/pull/40

Use <kbd>K</kbd> and <kbd>J</kbd> to move snippets up and down.

**Reply by wiredolphin** [reply]
Thank you for your answer. Anyway I already knew about this feature, which allows to reorder snippets. That is really helpful also. What I mean is rearrange the folders, that mean the first column entries on the left.

**Reply by maaslalani** [reply]
Ah I apologize for not reading properly! There is not currently a way to organize folders but definitely want something like this!
